.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+# When a GitHub release is created, this workflow will build and publish the executable.
+name: Release
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+'on':
+  release:
+    types:
+    - created
```

Comment:
So only manual release right? Would it be worth making all this automated on merges to main (including `cargo publish`)?

Reply:
I guess it depends on how often we make releases. If we wrote some automation like that, we could use it in recrypt-rs also.

Yes, the automation that's here only supports manual releases.